content/en/docs/reference/node/kubelet-checkpoint-api.md

```diff
@@ -25,6 +25,36 @@ should create the checkpoint archive to be only accessible by the `root` user. I
 is still important to remember if the checkpoint archive is transferred to another
 system all memory pages will be readable by the owner of the checkpoint archive.
 
+**Security Risks and Mitigation Strategies:**
```

Comment:
Please manually wrap the long lines where appropriate.

Reply:
check now

Reply:
Nothing changed.

Reply:
I m not able to understand exactly what you are saying